staff/aleix-caminal/cv/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+    <head>
+        <meta charset="utf-8">
+        <title>Curriculum Vitae</title>
+    </head>
+    <body>
+        <main>
+            <section id="head">
+                <div class="image">
```

Comment:
divs are not part of semantic html, may you use sections instead or other specific tags

Reply:
maybe an aside will work well here?